src/pages/home-page/Home.js

```diff
@@ -1,7 +1,8 @@
 import React from "react";
+import { Banner } from "../../components/components";
 
```

Comment:
suggestion - you can name the components.js file as index.js and then while import you only have to use "../../components"

Reply:
Done